tools/accuracy_checker/accuracy_checker/metrics/metric_profiler/base_profiler.py

```diff
@@ -74,24 +75,21 @@ def update(self, *args, **kwargs):
         profiling_data = self.generate_profiling_data(*args, **kwargs)
         self._last_profile = profiling_data
         if isinstance(profiling_data, list):
```

Comment:
I believe the list cases, report also contains list of records (e.g. each line represents the bounding box in detection report)